client/src/app/containers/Register/index.jsx

```diff
@@ -1,11 +1,39 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { useInjectReducer, useInjectSaga } from 'redux-injectors';
-import { Link, useHistory } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 import { authSaga } from '../../../auth/saga';
 import { sliceKey, reducer, actions } from '../../../auth/slice';
+import {
+  Button,
+  Container,
+  Link,
+  TextField,
+  CssBaseline,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 
-const Landing = () => {
+const useStyles = makeStyles(theme => ({
+  title: {
+    fontSize: '2rem',
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
```

Comment:
i am never supporting IE. I am late to the webdev game, and my code will not support it.

Reply:
This is straight from Google's example. What's the harm in leaving here, you curmudgeon? 😉